.github/workflows/student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative  'Person'
+
+class Student < Person
+    attr_reader :classroom
+
+    def initialize(name = "Unknown", age =nil, parent_permission = true, classroom)
+        super(name, age, parent_permission)
+        @classroom = classroom
+    end
+
+    def play_hooky
+        "¯\(ツ)/¯"
+    end
+def
```

Comment:
- [ ] [OPTIONAL] This class also has a syntax error. You are closing the `class` with a `def` instead of an `end` statement. Let's resolve this.

Reply:
Thanks for the insight.